specification/quantumleap.yml

```diff
@@ -566,6 +564,7 @@ paths:
             application/json:
               {
                 "data": {
+                    # Return type!?
```

Comment:
what's the problem here?

Reply:
it was just a mental note, to think if we shouldn't maybe return the NGSI type of the attr as well.

Reply:
the this should be commented with TODO so you can scout it easily